inappmessaging/app/src/main/java/com/google/firebase/example/inappmessaging/kotlin/MyClickListener.kt

```diff
@@ -18,3 +18,16 @@ class MyClickListener : FirebaseInAppMessagingClickListener {
     }
 }
 // [END fiam_click_listener]
+
+// [START fiam_click_listener_bundles]
+    override fun messageClicked(inAppMessage: InAppMessage, action: Action) {
```

Comment:
This will result in the same issue: can't have two of the same methods in the same class.  Maybe merged this into the existing messageClicked method?